recipes/tsebra/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "1.1.1" %}
+{% set name = "tsebra" %}
+{% set sha256 = "0426c243888135ee3fd6ea0ca9f37eb4af03dd117498165257fa756f59d49ce9" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/Gaius-Augustus/TSEBRA/archive/refs/tags/v.{{ version }}.tar.gz"
+  sha256: "{{ sha256 }}"
+  folder: tsebra 
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  build:
+    - python >=3.5.2
+  run:
+    - python >=3.5.2
+
+test:
+  commands:
+    - python tsebra.py --help
```

Comment:
This command tells python to run a local script at `./tsebra.py`. Since the scripts are now in the Conda environment path and have execute permissions, this line should be `tsebra.py --help`

Reply:
Thanks for your help